src/Helper/AbstractHtmlElement.php

```diff
@@ -72,7 +72,7 @@ protected function htmlAttribs($attribs)
         foreach ((array) $attribs as $key => $val) {
             $key = $escaper($key);
 
-            if (('on' == substr($key, 0, 2)) || ('constraints' == $key)) {
+            if ((0 === strpos($key, 'on')) || ('constraints' == $key)) {
```

Comment:
We can remove useless brackets here as well

Reply:
done ;)